pymc3/sampling.py

```diff
@@ -490,9 +490,9 @@ def sample(
                 progressbar=progressbar,
                 **kwargs,
             )
-            check_start_vals(start_, model)
             if start is None:
                 start = start_
+                check_start_vals(start, model)
```

Comment:
does `check_start_vals` not need to run here if `start` is not None?

Reply:
No, because that was already done at the beginning of the function (lines416-424)

Reply:
you're right, thanks!